src/DIRAC/Resources/Computing/AREXComputingElement.py

```diff
@@ -45,22 +45,6 @@ def __init__(self, ceUniqueID):
     def _reset(self):
         """Configure the Request Session to interact with the AREX REST interface.
         Specification : https://www.nordugrid.org/arc/arc6/tech/rest/rest.html
-
-        The following needed variables are obtained from the CS. If not available, some hopefully
-        sensible defaults are set.
-        "RESTEndpoint"      - CE is Queried if not available in CS
-               - The endpoint we talk to
-        "XRSLExtraString" - Default = ""
-               - Any CE specific string with additional parameters
-        "XRSLMPExtraString" - Default = ""
-               - Any CE specific string with additional parameters for MP jobs
-        "ARCRESTTimeout"    - DEfault = 1.0 (seconds)
-               - Timeout for the rest query
-        "proxyTimeLeftBeforeRenewal" - Default = 10000 (seconds)
-               - As the name says
-
-        Note : This is not run from __init__ as the design of DIRAC means that ceParameters is
-        filled with CEDefaults only at the time this class is initialised for the given CE
```

Comment:
Because many of these options are not defined in the method (`RESTEndpoint`, `XRSL(MP)ExtraString`) and we already have comments with the default values in `__init__`.